modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppStepContent.es.js

```diff
@@ -70,6 +74,29 @@ const EditAppStepContent = ({
 				{shortCutButton}
 			</ClayLayout.ContentRow>
 
+			<ClayLayout.ContentRow className="pl-4 pr-4">
+				{showInfoAlert && (
+					<ClayAlert
+						className="w-100"
+						displayType="info"
+						onClose={() => {
+							setShowInfoAlert(false);
+						}}
+						title={`${Liferay.Language.get('info')}:`}
+					>
+						{Liferay.Language.get(
+							'refresh-the-list-if-a-newly-created-workflow-is-not-displayed'
+						)}
+
+						<ClayLink
+							onClick={() => {
+								refetch();
+							}}
```

Comment:
Hey @naraandrade, could you just call the refetch function instead of create a new anonymous function to call this?

Reply:
I will do it, thanks for noticing @albinojunior . I'll pay more attention to this in the future.